### ISSUE TITLE
Unable to DELETE a serie with dash in name (400)

### ISSUE THREAD

**Issue by jd** [influxdb/influxdb#1967]
```
$ curl -X POST -d '[{"name":"12F9E1DF-71AB-494D-AE92-800EFA11B471","columns":["val"],"points":[[23]]}]' 'http://localhost:8086/db/gnocchi/series?u=root&p=root'
```
Then:

```
$ curl -v -X DELETE http://localhost:8086/db/gnocchi/series/12F9E1DF-71AB-494D-AE92-800EFA11B471\?u\=root\&p\=root
* Hostname was NOT found in DNS cache
*   Trying 127.0.0.1...
* Connected to localhost (127.0.0.1) port 8086 (#0)
> DELETE /db/gnocchi/series/12F9E1DF-71AB-494D-AE92-800EFA11B471?u=root&p=root HTTP/1.1
> User-Agent: curl/7.37.1
> Host: localhost:8086
> Accept: */*
>
< HTTP/1.1 400 Bad Request
< Access-Control-Allow-Headers: Origin, X-Requested-With, Content-Type, Accept
< Access-Control-Allow-Methods: GET, POST, PUT, DELETE
< Access-Control-Allow-Origin: *
< Access-Control-Max-Age: 2592000
< Content-Type: text/plain
< X-Influxdb-Version: InfluxDB v0.8.8 (git: afde71e) (leveldb: 1.18)
< Date: Mon, 16 Mar 2015 14:27:31 GMT
< Content-Length: 64
<
* Connection #0 to host localhost left intact
Error at 0:20 0:21. syntax error, unexpected '-', expecting $end
```

**Comment by jd** [comment]
Actually using a query returns the same error:

```
400: syntax error, unexpected '-', expecting $end
    drop series a5ad6ee8-b7bb-4a3d-98c5-fb315ef72e25
```

**Comment by pauldix** [comment]
This is fixed in 0.9.0.

**Comment by jd** [comment]
AFAICT this is not fixed in 0.9-rc12

I'm using the Python binding branch for 0.9 which sends ```DROP SERIES "myseries"``` and I get this:

```
400: {"error":"error parsing query: found \"5d2f5598-4030-4861-98c2-1f14170ab4cf\", expected number at line 1, char 13"}
```
